### ISSUE TITLE
allow "joined" syntax in key.json

### ISSUE THREAD

**Issue by MiniXC** [MEH-Design/atomicms#28]
Make
````
"/page": {
  template: "page",
  content: "page"
}
````
interchangeable with
````
"/page": "page"
````

**Comment by 1D0BE** [comment]
Test for this feature has been added to files
[Commit](https://github.com/MEH-Design/atomicms/commit/1224c77ca9f3f67d2e35285d02183a1ffbb3c6c9)

**Comment by bminixhofer** [comment]
resolved with commit a2b840c643e8703991b01b27595927d67b72a374
